plat/kvm/arm/setup.c

```diff
@@ -378,6 +378,23 @@ void init_secondary(uint64_t cpu)
 
 	if (smp_cpus == mp_ncpus)
 		smp_started = 1;
+
+	a = ukplat_memallocator_get();
+	if (a == NULL)
+		UK_CRASH("memallocator is not initialized\n");
+
+	s = uk_sched_default_init(a);
+	if (unlikely(!s))
+		UK_CRASH("Could not initialize the scheduler in APs\n");
+
+	/* Enable interrupts before starting the application */
+	ukplat_lcpu_enable_irq();
```

Comment:
I don't see where GIC CPU Interface was initialized for secondary cores..... actually GIC driver needs to be SMP aware, because it should initialize the distributor only once (from boot core), and then initialize CPU Interface for each core....